pkgs/development/python-modules/aiomysql/default.nix

```diff
@@ -18,6 +19,13 @@ buildPythonPackage rec {
     sha256 = "1qvy3phbsxp55161dnppjfx2m1kn82v0irc3xzqw0adfd81vaiad";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/aio-libs/aiomysql/commit/919b997a9de7f53d721af76762fba425e306531e.patch";
```

Comment:
Please add a comment what the patch accomplishes.

Reply:
would it be better to link to the patch such that it shows which fork the patch is from?

otherwise 
![image](https://user-images.githubusercontent.com/641278/117517204-42ad1b80-af69-11eb-802a-8ae0f6820c77.png)

Reply:
Which is why I suggested adding the link to the pull request.

Reply:
generally i think there's some concern about patches that appear as if they're from one repository, but are actually from a fork: https://github.com/NixOS/nixpkgs/issues/20836#issuecomment-626566603. that's not going on here, but we mind as well link to the proper fork. in the future, nixpkgs-hammering might flag this as a possible security issue: https://github.com/jtojnar/nixpkgs-hammering/pull/96

Reply:
What if fork removed, are patch links immutable? Otherwise I would prefer to keep link to original repo

Reply:
Is there a pull request that this commit is part of?